packages/web-components/fast-ssr/src/template-renderer/template-renderer.spec.ts

```diff
@@ -124,7 +125,51 @@ test.describe("TemplateRenderer", () => {
 
             expect(consolidate(result)).toBe(`<with-host-attributes  id="foo" static="static" dynamic="dynamic" bool-true><template shadowroot=\"open\">value<slot></slot></template></with-host-attributes>`);
         });
-    })
+    });
+    test.describe("rendering a FAST element configured with a null shadowOptions config", () => {
```

Comment:
Might be good to have a test that has `shadowOptions: null` and also has a template.

Reply:
oops I thought I added that, good catch